donate.html

```diff
@@ -17,7 +17,7 @@ <h1>Please Donate</h1>
     <div class="col-md-12 col-lg-8 offset-lg-2 text-right">
 
       <div class="form-group row">
-        <label for="BTC" class="col-sm-4 col-form-label"><strong>Bitcoin (BTC)</strong></label>
+        <label for="BTC" class="col-sm-4 col-form-label"><strong>Bitcoin (BTC)</strong> <a href="/assets/img/qr/BTC.png"><i class="fa fa-qrcode fa-2x"></i></a></label>
```

Comment:
```suggestion
        <label for="BTC" class="col-sm-4 col-form-label"><strong>Bitcoin (BTC)</strong> <a href="/assets/img/qr/BTC.png"><i class="fa fa-qrcode"></i></a></label>
```
![image](https://user-images.githubusercontent.com/3637842/55921414-3f94a500-5bc2-11e9-978d-7900e4152af9.png)